### ISSUE TITLE
SPFx VueJS "gulp bundle --ship" does not work

### ISSUE THREAD

**Issue by bratyslav** [pnp/generator-spfx#213]
#### General Information
* [ ] Usage
* Development
* [ ] Documentation
* [ ] Feature Request

#### Specific generator
* [ ] Addons
* [ ] HandlebarsJS
* VueJS

#### Version
* NodeJS: 10.16.3
* NPM: 6.13.1
* @pnp/spfx: 1.10.1

Hello. I created SPFx/VueJS project using your pnp/generator-spfx, and when I run command "gulp bundle --ship" I get some errors:

```
Error - [copy-assets] Webpack-produced asset "../lib/lib/webparts/eDocTask1/components/EDocTask1.vue.d.ts" does not appear to have a checksum in its filename, meaning that it is not safe to
 deploy to the CDN.
Error - [copy-assets] Webpack-produced asset "../lib/src/index.d.ts" does not appear to have a checksum in its filename, meaning that it is not safe to deploy to the CDN.
Error - [copy-assets] Webpack-produced asset "../lib/src/webparts/eDocTask1/EDocTask1WebPart.d.ts" does not appear to have a checksum in its filename, meaning that it is not safe to deploy
to the CDN.
```

How can I fix that?

P.S.: It is webpart for SP online

**Comment by StfBauer** [comment]
@bratyslav Just looked into your issue and it looks a bit strange. Currently, I wasn't able to figure out what caused this.

Is there any chance you can share your package.json and .yo-rc.json with me?

Thank you very much.

**Comment by StfBauer** [comment]
It seems like there is an issue in the build chain that causes your issues. I found, for now, a workaround that seems to work. I tested it on a web part and was able to deploy it to my tenant.

In order to make the build working, I changed the copy-assets.json to the following.

```json
{
  "$schema": "https://developer.microsoft.com/json-schemas/spfx-build/copy-assets.schema.json",
  "deployCdnPath": "temp/deploy",
  "extsToIgnore": ["vue", "d.ts", "d.ts.map"]
}
```

I adjust this in an upcoming release for all VueJS projects.

**Comment by StfBauer** [comment]
@bratyslav workaround is now implemented in the current version 1.11.0.
